csrc/multidevice/executor.h

```diff
@@ -116,10 +116,6 @@ class MultiDeviceExecutor {
   // execute a SegmentedGroup representing inter-device communication
   void postCommunication(SegmentedGroup* group);
 
-  // allocate inter-device communication recv buffers
-  std::unordered_map<Val*, c10::IValue> allocateRecvBuffers(
-      std::vector<c10::IValue> global_inputs_IValues);
-
   // Stores concrete computed values,
   std::unordered_map<Val*, c10::IValue> val_to_IValue_;
```

Comment:
(I understand this doesn't come from this PR.)

Why IValue instead of at::Tensor? Reading [its current usage](https://github.com/search?q=repo%3ANVIDIA%2FFuser%20val_to_ivalue_&type=code), `at::Tensor` seems to be good enough and probably simplifies (no need to toTensor etc).

Reply:
Thanks for the suggestion. I actually agree. I will open a new short PR to address that